docs/source/advanced/multi-target-ntypes.rst

```diff
@@ -1,12 +1,9 @@
-.. _advanced_usages:
-
-GraphStorm Advanced Usages
-===========================
+.. _multi_target_ntypes:
 
 Multiple Target Node Types Training
```

Comment:
For `Multiple Target Node Types Training`, I would recommend customers to use the multi-task learning framework instead of this solution.